docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1964,9 +1964,79 @@ CAUTION: Node role will take effect only with a 802.1X connection or if you use
 
 * Advanced filter under *Configuration -> Policies and Access Control -> Connection Profile -> Profile Name -> Advanced Filter*
 
-Examples
-
-* last_switch =~ "^JAMES" && extended.mse_inout.bob == "bobby"
+In this section you are able to define an advanced filter in order to match specific attributes.
+
+You have access to the following attributes:
+
+ From the database (mean that it comes from a previous connection):
+     node_info.autoreg
```

Comment:
'node_info.' is not used in the profile filter conditions

We can handle this in one of following ways
* Have the profile ```pf::factory::condition::profile``` remove 'node_info.' to allow consistency across filters
* Remove node_info. from the doc.